examples/arm/setup.sh

```diff
@@ -216,7 +216,7 @@ function setup_vela() {
     if [[ ! -e ethos-u-vela ]]; then
         git clone https://review.mlplatform.org/ml/ethos-u/ethos-u-vela
         repo_dir="${root_dir}/ethos-u-vela"
-        base_rev=d362f5443f67b1e6213a9d8f124edff758efac96
+        base_rev=fe0eaa55c5ed319f78c01978f3b40eb11a9bcb38
```

Comment:
In the summary can we add if there is any major change with this rev? Like one or two line would be helpful to understand the rationale.

Reply:
Yes, good idea!